### DISCUSSION TITLE
Forwarding to several users

### DISCUSSION THREAD

**Discussion by PlemGitHub** [aahnik/tgcf#268] [Q&A]
I need to process my referral system. My site drops service messages to Telegram with referral codes in it. I want to filter each message by such code and forward message to the specific person who owns that referral code. Can i do it?

So i have 1 source and many destinations, each of which must be with 1 specific filter.

**Comment by tissole** [comment]
The current script cannot do what you need because the config is global for all sources and destinations. But @aahnik has plans to make it work with a custom configuration for each source-destination, see #176.

Then it should work I think. If you know before the user name/ID, your chat name/ID, and the referral, you could configure each pair source-destination to send only messages that contains a specific referral (you put that in whitelist so only that would be forwarded).
